src/components/menu/menu.ts

```diff
@@ -339,9 +339,12 @@ export class Menu implements RootNode {
     }
 
     // normalize the "side"
-    if (this.side !== 'left' && this.side !== 'right') {
+    if (this.side !== 'left' && this.side !== 'right' && this.side !== 'start' && this.side !== 'end' && !_plt.isRTL()) {
       this.side = 'left';
     }
+    else if (this.side !== 'left' && this.side !== 'right' && this.side !== 'start' && this.side !== 'end' && _plt.isRTL()) {
+      this.side = 'right';
+    }
     this.setElementAttribute('side', this.side);
```

Comment:
This line will break the change. Let's say I have 'start', after construct is done, it will be `left`.

`this.side` should remain with the same value, then, wherever there is usage of `this.side` in the menu, it should be only `left` and `right`. Take a look at the function I wrote, to make it nicer.

* Note - this part of the code should not even exist (342-347). It should say:
```
if (['left', 'right', 'start', 'end'].indexOf(this.side) === -1) {
	this.side = 'start'; // default
	this.setElementAttribute('side', this.side);
}
```